.github/workflows/docs.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish docs
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/docs.yaml'
+      - 'docs/**'
+
+defaults:
+  run:
+    working-directory: docs
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '13'
+      - name: Install GitBook and build docs
+        run: |
+          npm install gitbook-cli
+          node_modules/gitbook-cli/bin/gitbook.js build
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs/_book
+          commit_message: "Update docs on push"
+          publish_branch: pages  # default: gh-pages
```

Comment:
this comment  here makes no sense to me

Reply:
It came copied from the configuration examples. Now I removed this line.